src/connections/ProjectPitch.ts

```diff
@@ -0,0 +1,132 @@
+import { createSelector } from "reselect";
+
+import { EntityIndexFilterKey } from "@/connections/Entity";
+import {
+  adminProjectPitchesIndex,
+  EntityIndexQueryParams,
+  projectPitchesGetUUIDIndex
+} from "@/generated/v3/entityService/entityServiceComponents";
+import { ProjectPitchDto } from "@/generated/v3/entityService/entityServiceSchemas";
+import {
+  adminProjectPitchesIndexFetchFailed,
+  adminProjectPitchesIndexIndexMeta,
+  projectPitchesGetUUIDIndexFetchFailed,
+  projectPitchesGetUUIDIndexIsFetching
+} from "@/generated/v3/entityService/entityServiceSelectors";
+import { getStableQuery } from "@/generated/v3/utils";
+import ApiSlice, { ApiDataStore, PendingErrorState, StoreResourceMap } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionLoader } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+const projectPitchesSelector = () => (store: ApiDataStore) =>
+  store["projectPitches"] as StoreResourceMap<ProjectPitchDto>;
+
+type ProjectPitchConnection = {
+  isLoading: boolean;
+  requestFailed: PendingErrorState | null;
+  isSuccess: boolean;
+  projectPitch: ProjectPitchDto | any;
+};
+
+type ProjectPitchConnectionProps = {
+  uuid: string;
+};
+
+const projectPitchConnection: Connection<ProjectPitchConnection, ProjectPitchConnectionProps> = {
+  load: ({ projectPitch }, { uuid }) => {
+    if (!projectPitch) projectPitchesGetUUIDIndex({ pathParams: { uuid } });
+  },
+
+  isLoaded: ({ projectPitch }) => projectPitch !== undefined,
+  selector: selectorCache(
+    ({ uuid }: ProjectPitchConnectionProps) => uuid,
+    ({ uuid }: ProjectPitchConnectionProps) =>
+      createSelector(
+        [
+          projectPitchesGetUUIDIndexIsFetching({ pathParams: { uuid } }),
+          projectPitchesGetUUIDIndexFetchFailed({ pathParams: { uuid } }),
+          projectPitchesSelector()
+        ],
+        (isLoading, requestFailed, selector) => ({
+          isLoading,
+          requestFailed,
+          isSuccess: selector?.organisationId != null,
+          projectPitch: selector[uuid]
+        })
+      )
+  )
+};
+
+export type ProjectsPitchesConnection = {
+  fetchFailure: PendingErrorState | null;
+  data?: ProjectPitchDto[];
+  indexTotal?: number;
+  refetch: () => void;
+};
+
+export type ProjectPitchIndexConnectionProps = {
+  pageSize?: number;
+  pageNumber?: number;
+  sortField?: string;
+  sortDirection?: "ASC" | "DESC";
+  filter?: Partial<Record<EntityIndexFilterKey, string>>;
+};
+
+const entityIndexQuery = (props?: ProjectPitchIndexConnectionProps) => {
```

Comment:
This should be `projectPitchIndexQuery` or something similar. It's a bit confusing, but in these contexts, "entity" specifically means projects / sites / nurseries / reports. ProjectPitch doesn't count as an "Entity"